barn/announcements-1.md

```diff
@@ -0,0 +1 @@
+Maintenance Alert: We’re moo-ving our orderbook to better pastures. Trading functionalities may be temporarily unavailable during this time.

```

Comment:
nit: Do we support markdown her (if so maybe we can make the **Maintenance Alert** part bold)?